cmd/veneur/main.go

```diff
@@ -81,6 +82,20 @@ func main() {
 		os.Exit(0)
 	}
 
+	stats, err := statsd.New(
+		config.StatsAddress,
+		statsd.WithAggregationInterval(config.Interval),
+		statsd.WithChannelMode(),
+		statsd.WithChannelModeBufferSize(4096),
+		statsd.WithClientSideAggregation(),
+		statsd.WithMaxMessagesPerPayload(4096),
+		statsd.WithNamespace("veneur."),
+		statsd.WithoutTelemetry(),
```

Comment:
Besides updating statsd client, this review also appears to do the following. Is that true?

- Modify WithAggregationInterval(config.Interval)
- Modify WithChannelMode()
- Modify WithChannelModeBufferSize(4096)
- Modify WithClientSideAggregation()

Reply:
If so, how can we test those changes? Can we update the review title/commit message to reflect the same?

Reply:
What you described is the only update to the statsd client. I added more to the PR title to indicate this.